src/compiler/types.ts

```diff
@@ -1393,18 +1394,19 @@ module ts {
     /* @internal */ 
     export const enum NodeCheckFlags {
         TypeChecked                 = 0x00000001,  // Node has been type checked
-        LexicalThis                 = 0x00000002,  // Lexical 'this' reference
-        CaptureThis                 = 0x00000004,  // Lexical 'this' used in body
-        EmitExtends                 = 0x00000008,  // Emit __extends
-        SuperInstance               = 0x00000010,  // Instance 'super' reference
-        SuperStatic                 = 0x00000020,  // Static 'super' reference
-        ContextChecked              = 0x00000040,  // Contextual types have been assigned
+        LexicalThis                 = 0x00000002,  // 'this' refers to a lexically captured '_this'.
+        CaptureThis                 = 0x00000004,  // 'this' needs to be captured as '_this'
+        CaptureArguments            = 0x00000008,  // 'arguments' needs to be captured as '_arguments'

```

Comment:
No, because unlike with `this`, you can't limit yourself to specific sites that need to be replaced with `_arguments` - if `arguments` needs to be captured, then *all* lexical `arguments` need to be replaced.